requirements.txt

```diff
@@ -27,6 +27,6 @@ s3transfer==0.3.3
 six==1.15.0
 SQLAlchemy==1.3.19
 urllib3==1.25.10
-uWSGI==2.0.19.1
+# uWSGI==2.0.19.1
```

Comment:
If you're going to comment out UWSGI to work on it locally, please don't commit it back.  If we merge as is, this line will break https://tasker.nvram.io.

Reply:
Ok.